.github/workflows/buf.yml

```diff
@@ -0,0 +1,42 @@
+name: Buf
+
+on:
+  pull_request:
+  push:
+    tags:
+      - v*
+    branches:
+      - develop
+      - release/v*
+      - feature/*
+
+jobs:
+  buf:
+    runs-on: ubuntu-20.04
+    if: "${{ !startsWith(github.event.head_commit.message, 'GitBook: [#') }}"
+    permissions:
+      contents: read
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup buf
+        uses: bufbuild/buf-setup-action@v1
+
+      - name: Lint protos
+        # TODO: Consider enabling this after fixing or ignoring the current linting errors?
+        if: false
```

Comment:
Linting is currently disabled because there are some linting errors already. If you want to enforce linting (according to the `buf` rules that is), we can enable this later.

Reply:
If will take a look but yes, I'm in for having proper linting enabled.